cli/compose/loader/loader_test.go

```diff
@@ -1324,16 +1326,23 @@ services:
     image: busybox
     extra_hosts:
       - "zulu:162.242.195.82"
+      - "whiskey=162.242.195.83"
       - "alpha:50.31.209.229"
       - "zulu:ff02::1"
-      - "host.docker.internal:host-gateway"
+      - "whiskey=ff02::2"
+      - "foxtrot=[ff02::3]"
```

Comment:
Do we support ipv6 addresses with  `[]` brackets and the `:` separator? If so we're missing a test for that